src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.NETCore.App": {
+            "type": "platform",
+            "version": "1.0.0-rc2-23931"
```

Comment:
No, if you are using NETStandard.Library in your project.json.pretemplate, you need to hard-code the version, since that version is for CoreFX.

Microsoft.NETCore.App uses the current CLI version.  CoreFX packages use a hard-coded version.